Source/ProgressionSystemRuntime/Private/LevelActors/PSStarActor.cpp

```diff
@@ -73,6 +73,7 @@ void APSStarActor::OnGameStateChanged_Implementation(ECurrentGameState GameState
 {
 	if (GameState == ECurrentGameState::Menu)
 	{
+		SetActorHiddenInGame(false); 
```

Comment:
Please try to find the root cause of this issue. If the issue is caused by the Pool Manager, then please leave comment and reassign the issue